src/Tests/dotnet.Tests/CommandTests/ToolInstallLocalCommandTests.cs

```diff
@@ -316,6 +322,122 @@ public void WhenRunWithPrereleaseAndPackageVersionItShouldSucceed()
             _fileSystem.File.Exists(restoredCommand.Executable.Value);
         }
 
+        [Fact]
+        public void GivenFeedVersionIsTheSameWhenRunWithPackageIdItShouldShowDifferentSuccessMessage()
+        {
+            GetDefaultTestToolInstallLocalCommand().Execute().Should().Be(0);
+            _reporter.Clear();
+            GetDefaultTestToolInstallLocalCommand().Execute().Should().Be(0);
+
+            AssertUpdateSuccess(packageVersion: _packageVersionA);
+            _reporter.Lines.Single()
+                .Should().Contain(
+                    string.Format(
+                        LocalizableStrings.UpdateLocaToolSucceededVersionNoChange,
+                        _packageIdA,
+                        _packageVersionA.ToNormalizedString(),
+                        _manifestFilePath));
+        }
+
+        [Fact]
+        public void GivenFeedVersionIsLowerRunPackageIdItShouldThrow()
+        {
+            GetDefaultTestToolInstallLocalCommand().Execute().Should().Be(0);
+            _mockFeed.Packages.Single().Version = "0.9.0";
```

Comment:
I think this scenario probably shouldn't fail.  If you run `dotnet tool install` and version 1.0.4 is already installed but the highest version in the feed is 0.9.0, then probably it should just leave 1.0.4 installed without failing (it should print a message letting you know it wasn't updated), the same as would happen if 1.0.4 was the highest version on the feed.  What do you think @baronfel?

There should be a separate test where if you have version 1.0.4 installed and the feed has 0.9.0 and 1.0.4 available, if you try to install and specify `--version 0.9.0`, then it should fail.

Reply:
Agree with @dsplaisted here.